src/main/resources/TestNGStandaloneTest.ftl

```diff
@@ -0,0 +1,14 @@
+package ${package};
+
+import org.testng.annotations.Test;
+import org.testng.Assert;
+import org.jboss.arquillian.testng.Arquillian;
```

Comment:
I think we should also include class under test ` import ${packageImport}.${ClassToTest};`